core-services/sanitize-prow-jobs/_config.yaml

```diff
@@ -0,0 +1,30 @@
+---
+defaultCluster: api.ci
+distribution:
+  api.ci:
+    jobs:
+    - apici-job1
+    - apici-job2
+    files:
```

Comment:
Why do we need this?

Reply:
want to express something like all periodics for the release repo have to be on api.ci